libs/ledger-live-common/src/__tests__/migration/account-migration.ts

```diff
@@ -0,0 +1,284 @@
+  "klaytn",
+  "polygon_zk_evm",
+  "polygon_zk_evm_testnet",
+  "base",
+  "base_sepolia",
+  "stacks",
+  "telos_evm",
+  "coreum",
+  "injective",
+  "casper",
+  "neon_evm",
+  "lukso",
+  "filecoin",
+  "linea",
+  "linea_goerli",
+]);
+
+LiveConfig.setConfig(liveConfig);
+LiveConfig.setAppinfo({
+  environment: "ci",
+  platform: "headless-linux",
+});
+
+const args = argv;
+
+type Args = {
+  /**
+   * comma seperated currencyId
+   * eg --currencies ethereum,polygon,bitcoin
+   */
+  currencies?: CryptoCurrencyId;
+  /**
+   * absolute path to the output folder for the json file
+   */
+  outputFolderPath?: string;
+  /**
+   * absolute path to the input json file
+   * must only contain an array of raw accounts
+   */
+  inputFile?: string;
+  /**
+   * if set, no file will be created
+   */
+  noEmit?: boolean;
+};
+
+const { currencies, inputFile, noEmit, outputFolderPath } = args as Args;
+
+const getMockAccount = (currencyId: string, address: string): Account => {
+  const currency = getCryptoCurrencyById(currencyId);
+
+  return {
+    name: "mockAccount",
+    type: "Account",
+    id: encodeAccountId({
+      type: "js",
+      version: "",
+      currencyId: currencyId,
+      xpubOrAddress: address,
+      derivationMode: "",
+    }),
+    freshAddress: address,
+    xpub: address,
+    derivationMode: "",
+    operations: [],
+    currency,
+    creationDate: new Date(0),
+    unit: currency.units[0],
+    balance: new BigNumber(0),
+    spendableBalance: new BigNumber(0),
+    blockHeight: 0,
+    freshAddressPath: "",
+    freshAddresses: [],
+    seedIdentifier: "",
+    index: 0,
+    starred: false,
+    used: true,
+    operationsCount: 0,
+    pendingOperations: [],
+    lastSyncDate: new Date(0),
+    balanceHistoryCache: {
+      HOUR: { latestDate: 0, balances: [] },
+      DAY: { latestDate: 0, balances: [] },
+      WEEK: { latestDate: 0, balances: [] },
+    },
+    swapHistory: [],
+  };
+};
+
+export const testSync = async (currencyId: string, xpubOrAddress: string) => {
+  console.log("starting sync on", currencyId, xpubOrAddress);
+  const mockAccount = getMockAccount(currencyId, xpubOrAddress);
+  const currency = getCryptoCurrencyById(currencyId);
+  const currencyBrige = getCurrencyBridge(currency);
+  const data = await currencyBrige.preload(currency);
+  currencyBrige.hydrate(data, currency);
+  const accountBrige = getAccountBridgeByFamily(mockAccount.currency!.family, mockAccount.id);
+
+  const syncedAccount = await firstValueFrom(
+    accountBrige
+      .sync(mockAccount, { paginationConfig: {}, blacklistedTokenIds: [] })
+      .pipe(reduce((acc, f: (arg0: Account) => Account) => f(acc), mockAccount)),
+  );
+
+  const accountRaw = toAccountRaw(syncedAccount);
+
+  console.log("finishing sync on", currencyId, xpubOrAddress);
+  return accountRaw;
+};
+
+(async () => {
+  if (inputFile && !existsSync(inputFile)) {
+    throw new Error(`Incorrect file path: ${inputFile} does not exist`);
```

Comment:
Do you think in that case we should log a warning and say that we only run sync because inputFile is provided but not available ?